torch/ao/quantization/quantize.py

```diff
@@ -524,6 +525,8 @@ def _convert(
                  is mutated
 
     """
+    if is_reference:
+        mapping = get_default_static_quant_reference_module_mappings()
```

Comment:
actually not like this, this should go within the L530 branch I think

Reply:
user should be able to overwrite the default mapping with explicit mappings, if user do not provide any mapping, we can provide a default mapping based on `is_reference` flag